core/src/main/scala/internal/NettySupport.scala

```diff
@@ -1,41 +1,20 @@
 package lol.http.internal
 
-import fs2.{ Stream, Strategy, Chunk, Task, Pull, Sink, async }
-
-import io.netty.channel.{
-  Channel,
-  ChannelFuture,
-  SimpleChannelInboundHandler,
-  ChannelHandlerContext }
-import io.netty.handler.logging.{ LogLevel, LoggingHandler }
-import io.netty.util.concurrent.{ GenericFutureListener }
-import io.netty.buffer.{ Unpooled, ByteBuf }
-import io.netty.handler.codec.http.{
-  HttpObject,
-  HttpContent,
-  LastHttpContent,
-  HttpMessage,
-  HttpResponse,
-  DefaultHttpContent,
-  DefaultHttpResponse,
-  DefaultHttpRequest,
-  DefaultLastHttpContent,
-  HttpClientCodec,
-  HttpResponseEncoder,
-  HttpRequestDecoder,
-  HttpContentDecompressor,
-  HttpMethod => NettyHttpMethod,
-  HttpVersion => NettyHttpVersion,
-  HttpResponseStatus,
-  HttpRequest,
-  HttpUtil }
-
-import scala.concurrent.{ Future, Promise }
-import scala.collection.mutable.{ ListBuffer }
+import scala.concurrent.ExecutionContext
+
+import cats._
+import cats.effect.IO
+import fs2.{Catenable, Chunk, Pull, Segment, Sink, Stream, async}
+import io.netty.channel.{Channel, ChannelFuture, ChannelHandlerContext, SimpleChannelInboundHandler}
+import io.netty.handler.logging.{LogLevel, LoggingHandler}
+import io.netty.util.concurrent.GenericFutureListener
+import io.netty.buffer.{ByteBuf, Unpooled}
+import io.netty.handler.codec.http.{DefaultHttpContent, DefaultHttpRequest, DefaultHttpResponse, DefaultLastHttpContent, HttpClientCodec, HttpContent, HttpContentDecompressor, HttpMessage, HttpObject, HttpRequest, HttpRequestDecoder, HttpResponse, HttpResponseEncoder, HttpResponseStatus, HttpUtil, LastHttpContent, HttpMethod => NettyHttpMethod, HttpVersion => NettyHttpVersion}
```

Comment:
Ugh, sorry, that line is embarrassingly long.  Must have been an IntelliJ auto-format that I didn't catch, since IntelliJ collapses all my imports.